### ISSUE TITLE
"Other Platforms" missing when editing game entry/Detect other games in Series

### ISSUE THREAD

**Issue by flexage** [TheGamesDB/TheGamesDB#38]
Reported by kente...@gmail.com, Dec 12, 2011
What steps will reproduce the problem?
1.in general View Game mode, Other Platform are visible

What is the expected output? What do you see instead?
Once you are in the "EDIT GAME" , the Other Platform 'tab' is not visible
The expected position towards the bottom of the listing with the Fanart/Screenshots

What version of the product are you using? On what operating system?
Win 7 

Please provide any additional information below.

Basically the Other Platform option is missing when editing a game.

flexage:
This particular position was intended to be reserved for an enhanced way of adding other games in a series with a linking function that auto-detects games that may match, and asks users to confirm/select the ones that do.
